### PR TITLE
[stable/influxdb]

### PR DESCRIPTION
**Pull request by aliartiza75**
#### Is this a new chart
No

#### What this PR does / why we need it:
This PR will add a table for config parameters.

#### Which issue this PR fixes
No issue

#### Special notes for your reviewer:
Kindly check the config parameter table.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [x] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by aliartiza75** [comment]
@jackzampolin kindly have a look at this PR.

**Comment by Aisuko** [comment]
Thanks for your work, there are lots of parameters in the `values.yaml` and may need a lot of time. If you wanna work on it, we could take a scope with the parameters which need to show on the `README.md`. If you wanna get any review, please signoff and pass the test case first.

**Comment by aliartiza75** [comment]
@Aisuko kindly share the list of params that should be added in the `README.md`

**Comment by aliartiza75** [comment]
@Aisuko tests have been passed

**Comment by Aisuko** [comment]
Sorry for replay late. In my opinion, the contents which in the chart' values.yaml should be show on the README.md and includes header name `Parameter(Name)`, `Description`, `Default(value)` and the description for the parameters we can follow the official site documents https://docs.influxdata.com/influxdb/v1.7/administration/config/

**Comment by aliartiza75** [comment]
Hi @Aisuko, 

I will do it as you suggested.

**Comment by Aisuko** [comment]
Thanks a lot.

**Comment by aliartiza75** [comment]
not stale

File: stable/influxdb/README.md [hunk 1]
```diff
@@ -45,6 +45,13 @@ The command removes all the Kubernetes components associated with the chart and
 
 The default configuration values for this chart are listed in `values.yaml`.
 
+#### General
```
Comment (naseemkullah) [inline stable/influxdb/README.md:48]:
```suggestion
The following table lists the configurable parameters of the InfluxDB chart and their default values.

```

File: stable/influxdb/README.md [hunk 2]
```diff
@@ -45,6 +45,13 @@ The command removes all the Kubernetes components associated with the chart and
 
 The default configuration values for this chart are listed in `values.yaml`.
```
Comment (naseemkullah) [inline stable/influxdb/README.md:46]:
```suggestion
```

File: stable/influxdb/README.md [hunk 3]
```diff
@@ -45,6 +45,13 @@ The command removes all the Kubernetes components associated with the chart and
...
 
+#### General
+
+| Parameter | Description | Default |
+|---|---|---|
+| image.repository | Image repository url | influxdb |
+| image.tag| Image tag | 1.7.6-alpine |
+
```
Comment (naseemkullah) [inline stable/influxdb/README.md:101]: We should expand this table with all values in values.yaml...

Comment (naseemkullah) [reply stable/influxdb/README.md:101]: @aliartiza75 Check out https://github.com/helm/charts/blob/master/stable/influxdb/values.yaml and include all those values please.

**Comment by aliartiza75** [comment]
Hi @Aisuko, i have updated the README.md file. kindly have a look at it and if it looks good kindly merge it.

File: stable/influxdb/README.md (hunk 3 above)
Comment (naseemkullah) [reply stable/influxdb/README.md:101]: 👏

**Comment by naseemkullah** [comment]
This is fantastic @aliartiza75 ... however we are in the process of moving the chart. Could you please open this PR in https://github.com/influxdata/helm-charts/tree/master/charts/influxdb ?Sorry about that. 😅

**Comment by aliartiza75** [comment]
@naseemkullah no problem , kindly merge it here. I will create another PR overthere.

**Comment by aliartiza75** [comment]
@Aisuko kindly remove the /hold tag/

**Review by Aisuko — APPROVED** [review]
/lgtm

**Comment by Aisuko** [comment]
@aliartiza75 Thanks for your patient.

**Pull request merged**